SimpleITK/utilities/resize_and_scale.py

```diff
@@ -0,0 +1,54 @@
+import SimpleITK as sitk
+
+
+def resize_and_scale(image, new_size, outside_pixel_value=0):
+    """
+    Resize the given image to the given size, with isotropic pixel spacing
+    and scale the intensities to [0,255].
+
+    Resizing retains the original aspect ratio, with the original image centered
+    in the new image. Padding is added outside the original image extent using the
+    provided value.
+
```

Comment:
Please add documentation that low-pass filtering should be done as a preprocessing step.